examples/motoko_examples/factorial/src/factorial.ts

```diff
@@ -0,0 +1,18 @@
+import {
+    Query,
```

Comment:
We should of course have linters and prettifiers do this for us, but I think a good convention is to alphabetize the imports